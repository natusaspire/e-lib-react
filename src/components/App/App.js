import React from 'react';

import './App.sass';
import Header from '@/components/Header';
import Main from '@/components/Main';
import Footer from '@/components/Footer';

const App = () => (
  <div className="wrapper">
    <div className="wrapper__header wrapper__header--fixed">
      <Header />
    </div>
    <div className="wrapper__main wrapper__main--grey">
      <Main />
    </div>
    <div className="wrapper__footer">
      <Footer />
    </div>
  </div>
);

export default App;
