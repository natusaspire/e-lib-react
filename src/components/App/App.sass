@import ~@/sass/wrapper
@import ~@/sass/header
@import ~@/sass/logo
@import ~@/sass/navbar
@import ~@/sass/main
@import ~@/sass/content
@import ~@/sass/grid
@import ~@/sass/card
@import ~@/sass/toolbar
@import ~@/sass/hero
@import ~@/sass/tabs
@import ~@/sass/buttons
@import ~@/sass/filters
@import ~@/sass/forms
@import ~@/sass/footer

html
  height: 100%
  margin: 0
  padding: 0
  border: 0

body
  height: 100%
  margin: 0
  padding: 0
  border: 0

#root
  height: 100%
  margin: 0
  padding: 0
  border: 0
