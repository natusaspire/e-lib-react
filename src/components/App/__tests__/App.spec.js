import React from 'react';
import { shallow } from 'enzyme';

import App from '@/components/App';

jest.mock('@/components/Header', () => 'Header');
jest.mock('@/components/Main', () => 'Main');
jest.mock('@/components/Footer', () => 'Footer');

describe('<App />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
