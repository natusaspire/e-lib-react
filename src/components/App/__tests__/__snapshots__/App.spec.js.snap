// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render app content 1`] = `
<div
  className="wrapper"
>
  <div
    className="wrapper__header wrapper__header--fixed"
  >
    <Header />
  </div>
  <div
    className="wrapper__main wrapper__main--grey"
  >
    <Main />
  </div>
  <div
    className="wrapper__footer"
  >
    <Footer />
  </div>
</div>
`;
